### DISCUSSION TITLE
Accessibility

### DISCUSSION THREAD

**Discussion by MireiaUB** [ChartsCSS/charts.css#53] [Q&A] [answered]
Great job with this library! nice features.

Could you integrate accessibility for low vision as well? for example with safe color combinations or patterns? that would make this library usable for public sites and for everyone conscious of inclusiveness.
Thanks!
Mireia Ribera at Universitat de Barcelona

**Comment by ramiy** [comment, marked as answer]
You can customize color and create your own theme with safe color combinations. https://chartscss.org/components/colors/

**Reply by MireiaUB** [reply]
That's good, but it would simplify things if some schemes were facilitated, such as in Vega:
https://vega.github.io/vega/docs/schemes/ 
and following colorblind-safe combinations, as those suggested by Cynthia Brewer:
https://colorbrewer2.org/#type=sequential&scheme=BuPu&n=6

If each author has to configure the color scales that would be a lot of work and probably not good elections.

Thanks!

Mireia Ribera, at Universitat de Barcelona
